### PR TITLE
Amélioration et optimisation.

### PR DESCRIPTION
**Pull request by lucguinchard**
Modification de la classe heletrope :
 - Utilisation de __CLASS__ au lieu de `heletrope`.
 - Ajout de la méthode createCmd.
 - Ajout de la méthode getLatitudeLongitude.
 - Suppression des méthodes date_sunrise et date_sunset.
 - Modification des LogicalId des commandes pour plus de clareté.
 - La recherche des heures de lever/coucher se fait lors du cron `plungins→cronDaily`. Soit à 00:00 et plus à 4:00.
 - Optimisations du code.

Modification dans la partie affichage des héliotropes et configuration :
 - Suppression du cron : la mise à jour se fait toutes les minutes à partir du cron `plugins→cron`, cela implique :
     - Suppression de la page de configuration.
     - Suppression de la page ajax.
 - Suppression des modals non utilisé.
 - Affichage des icônes non visible.
 - Modification de la présentation de la documentation (fond bleu, padding, utilisation de liste, mise au norme jeedomV4).

Modification dans la partie Template :
 - Les informations concernant l’information du jour arrivent sur la barre du panel.
 - Les Jauges s’adaptent à la largeur du panel.
 - Lors des mises à jour des données, le template n’est plus rafraîchit entièrement, cela implique :
    - Seules les données changées sont actualisées.
    - Les jauges ne se remettent pas à 0 lors des mises à jour.

### CODE REVIEW

**Comment by lunarok** [comment]
Désolé mais le commit est illisible tellement il y a de choses mélangées.
La doc n'est plus dans ce repo en v4, le fichier ajax doit rester par conformité Jeedom.
Renommer les commandes en anglais pourquoi pas mais trop d'impacts possible pour les user vu l'historique du plugin.
